rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -42,6 +42,20 @@ std::string to_string(T value)
 
 namespace rclcpp
 {
+
+/// Option to indicate what signal handlers should rclcpp install.
```

Comment:
```suggestion
/// Option to indicate which signal handlers rclcpp should install.
```

Reply:
Thanks! Fixed in https://github.com/ros2/rclcpp/pull/1771/commits/6c75ff7a28bcaef26b8e2df3426fb079a0172cc3.